db/change_cache.go

```diff
@@ -608,6 +609,11 @@ func (c *changeCache) processUnusedRange(ctx context.Context, fromSequence, toSe
 		// in pending.
 		base.WarnfCtx(ctx, "unused sequence range of #%d to %d contains duplicate sequences, will be ignored", fromSequence, toSequence)
 	}
+	if isEmpty {
+		if c.db.mutationListener.SkippedSequenceBroadcast.Load() {
+			c.db.mutationListener.SkippedSequenceBroadcast.Store(false)
+		}
```

Comment:
Can use `CompareAndSwap` here, since a separate atomic `Load` and `Store` will still allow for a race between the two lines from another goroutine.

CAS is implemented in one CPU instruction, so it's no more expensive than a `Load()` would be, even when successful.

```suggestion
		_ = c.db.mutationListener.SkippedSequenceBroadcast.CompareAndSwap(true, false)
```